marketplacetests/marketplace/regions/debug.py

```diff
@@ -21,6 +21,6 @@ def tap_back(self):
         self.marionette.find_element(*self._back_button_locator).tap()
 
     def select_region(self, region):
-        self.marionette.find_element(*self._region_select_locator).tap()
+        self.marionette.find_element(*self._region_select_locator).send_keys(region)
```

Comment:
Also the full viewport screenshots landed after the v2.0 branch was created, otherwise this might have been easier to debug (it would show the element was mostly off-screen).